src/audio/soundemitter.cpp

```diff
@@ -215,16 +218,37 @@ struct SoundOpenHandler : FileSystem::OpenHandler
 
 		ALenum alFormat = chooseALFormat(sampleSize, sample->actual.channels);
 
-		AL::Buffer::uploadData(buffer->alBuffer, alFormat, sample->buffer,
+		if (makeMono && sample->actual.channels == 2)
```

Comment:
hmm maybe this should be `> 1` rather than `== 2`? a sound technically can have more than two channels, right?